system/lib/compiler-rt/emscripten_exception_builtins.c

```diff
@@ -11,10 +11,10 @@
 
 #include <threads.h>
 
-thread_local int __THREW__ = 0;
+thread_local uintptr_t __THREW__ = 0;
```

Comment:
Perhaps add a comment here saying what value this can can have:

```
0 - Nothing thrown
1 - Exception thrown
Other values - jmpbuf pointer in the case that longjmp was thrown 
```

(BTW I think the doc in WebAssemblyLowerEmscriptenEHSjLj.cpp might be wrong.  It says that `__threwValue variable indicates the corresponding setjmp buffer the longjmp corresponds to`.  This seems to be wrong.. it looks like `__THREW__` hold the jmpbug pointer).


```
__threwValue - value passed to longjmp or ??? in the case of C++ exception.
```

Reply:
Done., and you're right that the description in LLVM is incorrect. Will fix there, thanks.